### DISCUSSION TITLE
[资讯]米哈游在多个平台停止了广告投放

### DISCUSSION THREAD

**Discussion by Dave-12138** [share121/inter-knot#1432] [绝区零]
在绝区零开服前，QQ音乐尚且还有绝区零的广告：
![Screenshot_20240701_133232_com tencent qqmusic](https://github.com/user-attachments/assets/3018cd3e-992f-46ff-9bc9-9c384fbd761e)
但如今已经一个米游广告都没有了，类似的情况还出现在QQ空间。
（但手机广告配的测评视频还是原神）
想来米哈游的推广也转向存量市场了。

**Comment by lllliufeng** [comment]
我edge上还有ww
![image](https://github.com/user-attachments/assets/ff93454c-5841-49d7-9861-9bff27109db9)
